src/com/reco1l/osu/beatmaplisting/BeatmapListing.kt

```diff
@@ -568,13 +614,18 @@ class BeatmapSetViewHolder(itemView: View, private val mediaScope: CoroutineScop
                     if (BeatmapListing.isPlayingMusic) {
                         GlobalManager.getInstance().mainScene.musicControl(MusicOption.PLAY)
                     }
+
+
```

Comment:
So far so good. Nitpicking the formatting here but the line breaks here are unnecessary.